neuralpredictors/layers/readouts.py

```diff
@@ -944,7 +944,7 @@ def initialize_shared_grid(self, match_ids=None, shared_grid=None):
         self.register_buffer("grid_sharing_index", torch.from_numpy(sharing_idx))
         self._shared_grid = True
 
-    def forward(self, x, sample=None, shift=None, out_idx=None):
+    def forward(self, x, sample=None, shift=None, out_idx=None, multiplex=None, crop_edge_px=None):
```

Comment:
Let `multiplex` default to `False`, and let's also add a new argument to control whether the output gets collapsed (e.g. `collapse=True`)